hack/release.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash -e
 
+make IMAGE_TAG=$TAG docker-build docker-push
```

Comment:
isn't this redundant ? we also do it in the travis.yml

Reply:
This is for release what we have at .travis.yaml after this PR is at mater

Reply:
yeah sorry total brain-fart